pkg/cli/scan.go

```diff
@@ -29,13 +29,12 @@ func Scan() *cobra.Command {
 					return fmt.Errorf("failed to open apk file: %w", err)
 				}
 
-				fmt.Println(path.Base(apkFilePath))
+				fmt.Printf("Will process: %s\n", path.Base(apkFilePath))
```

Comment:
Can we make sure this gets written to stderr instead of stdout?

Reply:
(we'll want to make sure only the scan output goes to stdout, esp when we support a machine-readable output)

Reply:
```
$ ./wolfictl scan ~/Downloads/os_x86_64_bash-doc-5.2_rc4-r0.apk
Will process: os_x86_64_bash-doc-5.2_rc4-r0.apk
✅ No vulnerabilities found

$ ./wolfictl scan ~/Downloads/os_x86_64_bash-doc-5.2_rc4-r0.apk 2>/dev/null
✅ No vulnerabilities found
```